utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -35,6 +36,8 @@ import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.ort.execute
 
 private const val HTTP_CLIENT_CONNECT_TIMEOUT_IN_SECONDS = 30L
+private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 5 * 60L
```

Comment:
Commit message: The "While at it," part as if you're doing another thing at the same time, but in fact passing the default constructor arguments explicitly *is* the way to stop sharing the connection pool. So I'd simply say "Stop sharing the connection pool with HTTP clients used elsewhere by passing the default constructor parameter values explicitly". Also, `IOException` should be put in backticks as well.

Reply:
> The "While at it," part as if you're doing another thing at the same time

I wrote "while at it" because I'm actually doing two things at the same time:

1. Using a separate connection pool
2. passing default constructor arguments explicitly

...because 1 could also be done without 2. 
Anyhow, changing the commit message to how you want it to be.